patches/api/0126-PlayerLaunchProjectileEvent.patch

```diff
@@ -67,6 +68,7 @@ index 0000000000000000000000000000000000000000..9074b2ede01f76c0560e531824638216
 +
 +    /**
 +     * Set whether to consume the ItemStack or not
++     * Note: this does nothing for reusable and throwable item like the trident
```

Comment:
Is there any reason we cannot actually implement this ?
I mean, the item removal logic can technically just be skipped if the flag is set no ?

Reply:
Yes i forgot why i have made that, but i think the main idea was avoid a duplicate item in survival but since it's plugin anyway, this can be cool for mini game. will implement that

Reply:
Yea, I figured that was the concept but I agree with you here. "Should consume" set to false directly means duplication tbh.
Yea it is an item here, it might as well be xp from bottles.

Reply:
The only question left is should we avoid the hurt damage on the item when this is disabled or not

Reply:
Well, this does indeed pose an interesting question.
I'd presume we would want to leave the item undamaged as this is as close as possible to the "just don't consume whatever was used, only launch" mentality at least I think of in regards to the event.